Modules/Devices/src/Devices.ConsumerApi/Controllers/IdentitiesController.cs

```diff
@@ -59,16 +59,16 @@ public async Task<IActionResult> CreateIdentity(CreateIdentityRequest request, C
 
 public class CreateIdentityRequest
 {
-    public string ClientId { get; set; }
-    public string ClientSecret { get; set; }
-    public byte[] IdentityPublicKey { get; set; }
-    public string DevicePassword { get; set; }
+    public required string ClientId { get; set; }
+    public required string ClientSecret { get; set; }
+    public required byte[] IdentityPublicKey { get; set; }
+    public required string DevicePassword { get; set; }
     public byte IdentityVersion { get; set; }
```

Comment:
Why didn't you make this one required?

Reply:
Overlooked it because though ReSharper didn't flag it as a problem actually.